packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -985,6 +985,11 @@ describe('src/cy/commands/querying', () => {
       })
     })
 
+    // https://github.com/cypress-io/cypress/issues/25025
+    it('searches multiple subject elements', () => {
+      cy.get('ul').contains('li', 'asdf 3')
+    })
```

Comment:
to ensure this change touched within subject, can we we add something like this as well?
```js
    it('searches multiple subject elements in .within', () => {
      cy.get('body').within(() => {
         cy.get('ul').contains('li', 'asdf 3')
       })
    })
```

Reply:
I believe this case is already well covered inside the `.within()` tests.

```
it('scopes additional CONTAINS finders to the subject', () => {
      const span = cy.$$('#nested-div span:contains(foo)')

      cy.contains('foo').then(($span) => {
        expect($span.get(0)).not.to.eq(span.get(0))
      })

      cy.get('#nested-div').within(() => {
        cy.contains('foo').then(($span) => {
          expect($span.get(0)).to.eq(span.get(0))
        })
      })
    })
```

and a couple of other similar tests in within.cy.js I don't think it's valuable to make this test more complicated.

Reply:
@BlueWinds wasn't my suggestion for a test this issue? https://github.com/cypress-io/cypress/issues/25019

Reply:
I hadn't figured that out when I made this PR, was taking them one at a time. But that other PR has its own test covering that case. `.within()` isn't necessary to hit either of these two bugs - they're entirely within .contains(), users just happened to run into them using `.within()` blocks.

Reply:
I had a lazy check that was looking for `subject[0]` as equivelent for "is a jquery object", which worked for every element... except `<form>`. Today I learned.

Reply:
The fact that .contains() hijacks the "withinSubject" property of .get() was pre-existing behavior that I really should have cleaned up / renamed when I moved them to be queries, but didn't think of at the time. It's really more a "container element to query inside" - which we call "withinSubject" because .within() is our "scope commands to a container element" command.